atmega328/sonoffsc/sonoffsc.ino

```diff
@@ -39,9 +41,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define DHT_PIN                 6
 #ifndef DHT_TYPE
-#define DHT_TYPE                DHT11
+#define DHT_TYPE                DHT22
```

Comment:
Fair point, might be worth having both in (one commented) to make it easy for beginners to make the required choice.